src/core/cards/card.ts

```diff
@@ -123,6 +135,11 @@ export abstract class Card {
   public is(type: CardType) {
     return this.cardType.includes(type);
   }
+
+  public isCommonTrick() {
```

Comment:
I'd suggest to return false in the base class, and override it in `TrickCard` class